Swift interop tests fail to build for NativeAOT

Running `src/tests/build.sh -nativeaot -tree:Interop /p:LibrariesConfiguration=Debug` on macOS is currently broken due to the new Swift interop tests:

```
    Unhandled exception: System.AggregateException: One or more errors occurred. (Code generation failed for method '[SwiftInvalidCallConv]InvalidCallingConvTests.FuncWithSIMDArg(Vector4)') (Code generation failed for method '[SwiftInvalidCallConv]InvalidCallingConvTests.FuncWithSwiftErrorAsArg(SwiftError)')
     ---> ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftInvalidCallConv]InvalidCallingConvTests.FuncWithSIMDArg(Vector4)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0
       --- End of inner exception stack trace ---
       at System.Threading.Tasks.TaskReplicator.Run[TState](TaskReplicator.ReplicatableUserAction`1, ParallelOptions, Boolean) + 0x1bc
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x43c
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.ThrowSingleCancellationExceptionOrOtherException(ICollection, CancellationToken, Exception) + 0x64
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x530
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IList`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x160
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x154
       at System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable`1, ParallelOptions, Action`1) + 0x98
       at ILCompiler.RyuJitCompilation.CompileMultiThreaded(List`1) + 0x164
       at ILCompiler.RyuJitCompilation.ComputeDependencyNodeDependencies(List`1 obj) + 0x250
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeDependencies(List`1) + 0x58
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeMarkedNodes() + 0xe8
       at ILCompiler.RyuJitCompilation.CompileInternal(String, ObjectDumper) + 0x4c
       at ILCompiler.Compilation.ILCompiler.ICompilation.Compile(String, ObjectDumper) + 0x74
       at ILCompiler.Program.Run() + 0x32c4
       at ILCompiler.ILCompilerRootCommand.<>c__DisplayClass236_0.<.ctor>b__0(ParseResult result) + 0x390
       at System.CommandLine.Invocation.AnonymousSynchronousCliAction.Invoke(ParseResult) + 0x38
       at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult) + 0x190
     ---> (Inner Exception #1) ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftInvalidCallConv]InvalidCallingConvTests.FuncWithSwiftErrorAsArg(SwiftError)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0<---
    
/Users/filipnavara/Projects/runtime/artifacts/bin/coreclr/osx.arm64.Debug/build/Microsoft.NETCore.Native.targets(310,5): error MSB3073: The command ""/Users/filipnavara/Projects/runtime/artifacts/bin/coreclr/osx.arm64.Debug/ilc-published/ilc" @"/Users/filipnavara/Projects/runtime/artifacts/tests/coreclr/obj/osx.arm64.Debug/Managed/Interop/Swift/SwiftInvalidCallConv/SwiftInvalidCallConv/native/SwiftInvalidCallConv.ilc.rsp"" exited with code 1. [/Users/filipnavara/Projects/runtime/src/tests/Interop/Swift/SwiftInvalidCallConv/SwiftInvalidCallConv.csproj] [/Users/filipnavara/Projects/runtime/src/tests/build.proj]
    Unhandled exception: System.AggregateException: One or more errors occurred. (Code generation failed for method '[SwiftErrorHandling]ErrorHandlingTests.conditionallyThrowError(int32,SwiftError&)') (Code generation failed for method '[SwiftErrorHandling]ErrorHandlingTests.conditionallyThrowErrorOnStack(int32,int32,int32,int32,int32,int32,int32,int32,int32,int32,SwiftError&)')
     ---> ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftErrorHandling]ErrorHandlingTests.conditionallyThrowError(int32,SwiftError&)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0
       --- End of inner exception stack trace ---
       at System.Threading.Tasks.TaskReplicator.Run[TState](TaskReplicator.ReplicatableUserAction`1, ParallelOptions, Boolean) + 0x1bc
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x43c
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.ThrowSingleCancellationExceptionOrOtherException(ICollection, CancellationToken, Exception) + 0x64
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x530
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IList`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x160
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x154
       at System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable`1, ParallelOptions, Action`1) + 0x98
       at ILCompiler.RyuJitCompilation.CompileMultiThreaded(List`1) + 0x164
       at ILCompiler.RyuJitCompilation.ComputeDependencyNodeDependencies(List`1 obj) + 0x250
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeDependencies(List`1) + 0x58
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeMarkedNodes() + 0xe8
       at ILCompiler.RyuJitCompilation.CompileInternal(String, ObjectDumper) + 0x4c
       at ILCompiler.Compilation.ILCompiler.ICompilation.Compile(String, ObjectDumper) + 0x74
       at ILCompiler.Program.Run() + 0x32c4
       at ILCompiler.ILCompilerRootCommand.<>c__DisplayClass236_0.<.ctor>b__0(ParseResult result) + 0x390
       at System.CommandLine.Invocation.AnonymousSynchronousCliAction.Invoke(ParseResult) + 0x38
       at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult) + 0x190
     ---> (Inner Exception #1) ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftErrorHandling]ErrorHandlingTests.conditionallyThrowErrorOnStack(int32,int32,int32,int32,int32,int32,int32,int32,int32,int32,SwiftError&)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0<---
    
/Users/filipnavara/Projects/runtime/artifacts/bin/coreclr/osx.arm64.Debug/build/Microsoft.NETCore.Native.targets(310,5): error MSB3073: The command ""/Users/filipnavara/Projects/runtime/artifacts/bin/coreclr/osx.arm64.Debug/ilc-published/ilc" @"/Users/filipnavara/Projects/runtime/artifacts/tests/coreclr/obj/osx.arm64.Debug/Managed/Interop/Swift/SwiftErrorHandling/SwiftErrorHandling/native/SwiftErrorHandling.ilc.rsp"" exited with code 1. [/Users/filipnavara/Projects/runtime/src/tests/Interop/Swift/SwiftErrorHandling/SwiftErrorHandling.csproj] [/Users/filipnavara/Projects/runtime/src/tests/build.proj]
/Users/filipnavara/Projects/runtime/src/tests/Interop/NativeLibrary/NativeLibraryToLoad/NativeLibraryToLoad.cs(49): warning IL3000: NativeLibraryToLoad.GetDirectory(): 'System.Reflection.Assembly.Location.get' always returns an empty string for assemblies embedded in a single-file app. If the path to the app directory is needed, consider calling 'System.AppContext.BaseDirectory'. [/Users/filipnavara/Projects/runtime/src/tests/Interop/NativeLibrary/Callback/CallbackStressTest_TargetUnix.csproj] [/Users/filipnavara/Projects/runtime/src/tests/build.proj]
    Unhandled exception: System.AggregateException: One or more errors occurred. (Code generation failed for method '[SwiftSelfContext]SelfContextTests.getMagicNumberOnStack(int32,int32,int32,int32,int32,int32,int32,int32,int32,int32,SwiftSelf)') (Code generation failed for method '[SwiftSelfContext]SelfContextTests.getMagicNumber(SwiftSelf)')
     ---> ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftSelfContext]SelfContextTests.getMagicNumberOnStack(int32,int32,int32,int32,int32,int32,int32,int32,int32,int32,SwiftSelf)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0
       --- End of inner exception stack trace ---
       at System.Threading.Tasks.TaskReplicator.Run[TState](TaskReplicator.ReplicatableUserAction`1, ParallelOptions, Boolean) + 0x1bc
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x43c
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.ThrowSingleCancellationExceptionOrOtherException(ICollection, CancellationToken, Exception) + 0x64
       at System.Threading.Tasks.Parallel.ForWorker[TLocal,TInt](TInt, TInt, ParallelOptions, Action`1, Action`2, Func`4, Func`1, Action`1) + 0x530
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IList`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x160
       at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable`1, ParallelOptions, Action`1, Action`2, Action`3, Func`4, Func`5, Func`1, Action`1) + 0x154
       at System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable`1, ParallelOptions, Action`1) + 0x98
       at ILCompiler.RyuJitCompilation.CompileMultiThreaded(List`1) + 0x164
       at ILCompiler.RyuJitCompilation.ComputeDependencyNodeDependencies(List`1 obj) + 0x250
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeDependencies(List`1) + 0x58
       at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeMarkedNodes() + 0xe8
       at ILCompiler.RyuJitCompilation.CompileInternal(String, ObjectDumper) + 0x4c
       at ILCompiler.Compilation.ILCompiler.ICompilation.Compile(String, ObjectDumper) + 0x74
       at ILCompiler.Program.Run() + 0x32c4
       at ILCompiler.ILCompilerRootCommand.<>c__DisplayClass236_0.<.ctor>b__0(ParseResult result) + 0x390
       at System.CommandLine.Invocation.AnonymousSynchronousCliAction.Invoke(ParseResult) + 0x38
       at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult) + 0x190
     ---> (Inner Exception #1) ILCompiler.CodeGenerationFailedException: Code generation failed for method '[SwiftSelfContext]SelfContextTests.getMagicNumber(SwiftSelf)'
     ---> System.NotImplementedException: canGetCookieForPInvokeCalliSig
       at Internal.JitInterface.CorInfoImpl.canGetCookieForPInvokeCalliSig(CORINFO_SIG_INFO*) + 0x40
       at Internal.JitInterface.CorInfoImpl._canGetCookieForPInvokeCalliSig(IntPtr, IntPtr*, CORINFO_SIG_INFO*) + 0x60
       --- End of inner exception stack trace ---
       at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode, MethodIL) + 0x21c
       at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodCodeNode, MethodIL) + 0xc0
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(CorInfoImpl, MethodCodeNode) + 0xb8
       at ILCompiler.RyuJitCompilation.CompileSingleMethod(MethodCodeNode methodCodeNodeNeedingCode) + 0x84
       at System.Threading.Tasks.Parallel.<>c__DisplayClass32_0`2.<ForEachWorker>b__0(Int32 i) + 0x70
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x2b4
    --- End of stack trace from previous location ---
       at System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`2.<ForWorker>b__1(RangeWorker& currentWorker, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion) + 0x578
       at System.Threading.Tasks.TaskReplicator.Replica`1.ExecuteAction(Boolean&) + 0x50
       at System.Threading.Tasks.TaskReplicator.Replica.Execute() + 0xa0<---
 ```

--- a/src/tests/issues.targets
+++ b/src/tests/issues.targets
@@ -1158,6 +1158,10 @@
         <ExcludeList Include="$(XunitTestBinBase)/Loader/CustomAttributes/**">
             <Issue>Dynamic code generation is not supported on this platform</Issue>
         </ExcludeList>
+
+        <ExcludeList Include="$(XunitTestBinBase)/Interop/Swift/**">
+            <Issue>https://github.com/dotnet/runtime/issues/93631</Issue>
+        </ExcludeList>
     </ItemGroup>
 
     <!-- run.proj finds all the *.cmd/*.sh scripts in a test folder and creates corresponding test methods.

